AutoClicker/Views/MainWindow.xaml.cs

```diff
@@ -290,6 +315,11 @@ private void OnClickTimerElapsed(object sender, ElapsedEventArgs e)
                     clickTimer.Stop();
                     ResetTitle();
                 }
+
+                var interval = CalculateInterval();
+                if (IsIntervalValid())
+                    clickTimer.Interval = CalculateInterval();

```

Comment:
A third interval calculation? Why not use the `interval `variable from above?

Reply:
There is some duplicate calculation is the current PR which is one of the things that needs to be cleaned up and is part of the reason why the PR is a draft.

Reply:
Should be resolved